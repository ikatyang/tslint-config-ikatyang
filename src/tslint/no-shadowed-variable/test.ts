// good

let bad1 = 1;

(function () {
  let bad2 = 1;
})();


// bad

let bad1 = 1;

(function () {
  let bad1 = 1;
})();
