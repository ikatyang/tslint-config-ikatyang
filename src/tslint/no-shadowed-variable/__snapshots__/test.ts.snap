// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

let bad1 = 1;

(function () {
  let bad2 = 1;
})();


// bad

let bad1 = 1;

(function () {
  let bad1 = 1;
      ~~~~      [Shadowed name: 'bad1']
})();

`;
